CHANGELOG.md

```diff
@@ -99,6 +99,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Modify/delete conflicts now include context lines
```

Comment:
Minor re commit message: I'd make the title "conflicts: Fix bug when modifying edit-delete conflicts".

Reply:
Done